src/contracts/aludel.ts

```diff
@@ -52,6 +56,139 @@ export async function getNetworkStats(signer: any) {
+  const rewardScalingPeriod = 60 * 24 * 60 * 60;
+  const floor = 1;
+  const ceiling = 10;
+  const now = Math.floor(Date.now() / 1000); // Replace with staking period start
+  // Total rewards pool
+  let [mistRewardsBalance, wethRewardsBalance, aludelData] = await Promise.all([
+    bonusMistToken.balanceOf(args.rewardPool), //returns BN
+    wethRewardToken.balanceOf(args.rewardPool), //returns BN
+    aludel.getAludelData(), // 5th or 7th arg
+  ])
+  console.log("Aludel data", aludelData)
+
+  let chartData = [];
+    // Get rewards and stake units at future time (days elapsed)
+  let sharesOutstanding = aludelData[4]
+  let currentTotalStakeUnits = aludelData[6]
+  let currentTotalStake = aludelData[5]
+  let scheduleShares = aludelData[8][0][2] // assumes 1 schedule
+  let lastUpdate = now;
+
+  for (let day = 0; day <= 60; day += 3) {
+    const daysAdditionalInSecs =  day * 60 * 60 * 24;
+    const stakeDuration = now + daysAdditionalInSecs - stakingStart; 
+    const totalUnlockedWeiRewards = getFutureUnlockedRewards(daysAdditionalInSecs, wethRewardsBalance, sharesOutstanding, scheduleShares); //returns BN
+    currentTotalStakeUnits = getFutureTotalStakeUnits(currentTotalStakeUnits, currentTotalStake, now + daysAdditionalInSecs - lastUpdate)
+    lastUpdate = now + daysAdditionalInSecs
+    // total stake units = total stake amount * total duration
+    // const totalStakeUnits = FixedNumber.from(currentTotalStake).mulUnsafe(FixedNumber.from(now  + daysAdditionalInSecs - inflationStart));
+
+    // Stake units = LP token amount * user stake duration
+    let userStake = FixedNumber.from(amount).mulUnsafe(FixedNumber.from(now  + daysAdditionalInSecs - stakingStart)) 
+
+    console.log("User Stake", userStake.toString())
+    console.log("Total Stake", aludelData[5].toString())
+    console.log("Total Stake Units", currentTotalStakeUnits.toString())
+    console.log("Total Unlocked Rewards", totalUnlockedWeiRewards.toString(), currentTotalStakeUnits.toString())
+    console.log("Seconds Elapsed", daysAdditionalInSecs.toString())
+    console.log("Stake Duration", stakeDuration)
+    
+    // total unlocked wei * user stake / total stake 
+    let weiRewards = totalUnlockedWeiRewards.mulUnsafe(userStake).divUnsafe(FixedNumber.from(currentTotalStakeUnits)); // creates ratio of pool owned
+    console.log("Wei rewards before multiplier", weiRewards.toString())
+
+    if (stakeDuration <= 5184000) {
+      console.log("less than 60 days")
+      weiRewards = weiRewards
+        .mulUnsafe(FixedNumber.from(stakeDuration))
+        .divUnsafe(FixedNumber.from(rewardScalingPeriod))
+        .mulUnsafe(FixedNumber.from(ceiling - floor)) // 9
+        .divUnsafe(FixedNumber.from(ceiling)); // 10
+    }
+
+    console.log("Wei Rewards after multiplier", weiRewards.toString());
+
+    // Calculate inflation bonuses to mist rewards pools
+    const inflationStart = 1612643118;
```

Comment:
`inflationStart` is about when the contract was created. Not sure if this is right.